### ISSUE TITLE
1.15.2 writers-lang-and-dir tests fail due to missing files

### ISSUE THREAD

**Issue by juhp** [jgm/pandoc#2537]
When running the testsuite for 1.15.2 in the Stackage Nightly build:

```
:
    asciidoc:
      basic: [OK]
      tables: [OK]
  writers-lang-and-dir:
    latex: [Failed]
ERROR: writers-lang-and-dir.latex: openBinaryFile: does not exist (No such file or directory)
    context: [Failed]
ERROR: writers-lang-and-dir.context: openBinaryFile: does not exist (No such file or directory)
Shared:
  normalize:
    p_normalize_blocks_rt: [OK, passed 100 tests]
:
```

**Comment by juhp** [comment]
Thank you!

**Comment by jgm** [comment]
As you may have seen, I released 1.15.2.1 which fixes this.

+++ Jens Petersen [Nov 17 15 02:07 ]:
